ICA/auto_mycosnp_ICA.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date
+
+# This funciton uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinate will keep script running as long as it dosent error out.
+go=1
+while go == 1:
+# Sequencing runs are directly upload to ICA after basecalling into this project
+    bashCommand="icav2 projects enter BSSH UPHL_Bioinfo"
+    process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+    process.communicate()
+#Folders of ICA Runs
+    bashCommand = "icav2 projectdata list --data-type FOLDER --file-name app.* --match-mode FUZZY"
+    tmp= icav_out(bashCommand)
+# ICA runs
+    ica_r=[]
+    for i in tmp:
+        ica_r.append(i.split(" ")[0])
+
+    runs_on_ica=[]
+    for i in range(len(tmp)):
+        runs_on_ica.append(tmp[i].split()[0].split('/')[2])
+# The plan is to download only the files that are commonly needed by Tom and use these directories as a way to way to moniter if anaylsis has completed.
+    runs_completed_analysis=os.listdir("/Volumes/IDGenomics_NAS/ICA/Runs_On_ICA")
```

Comment:
Can you add a glob that could be something like
```
import glob
runs_completed_analysis=glob("/Volumes/IDGenomics_NAS/ICA/Runs_On_ICA/UT*")
```

Reply:
Do we want all Granduer runs to run on ICA or the linux work station? I was thinking we could change General_LW_scripts/MiSeq_runs_basespace.sh to not run Granduer on the work station once we are confident in ICA.

Reply:
Once CLIA has been approved to run Grandeur on ICA, all of them will be run on ICA. It has to be approved by our CLIA officer before we make the switch, though.

Reply:
The entire grandeur directory would be ideal. 

I don't mind if it's compressed before download.

Reply:
Here are the mycosnp_ICA results files I'll need downloaded. 

/results/stats/qc_report/qc_report.txt
/results/multiqc/multiqc_report.html
/results/combined/vcf-to-fasta/vcf-to-fasta.fasta
/results/combined/snpdists/combined.tsv

Reply:
@tives82 , you need the fastq files, too, right?

Reply:
For the CLIA validation some of those folders were 100 GB and took about 2 hrs to download. I am reaching out to ICA to see if I can compress those files on there platform and then download them.  They will probably tell me to use bench but thats costs compare to the just dealing with the time that I don't think it would be worth it.

Reply:
@erinyoung, Yeah, I'll need the fastq files downloaded as well. 
@jwarnn, I know snpdists wasn't one of the files we had talked about previously but I figure it's a good idea to have the snpdists file be downloaded due to more Epis requesting it and the ability for us to upload it into Microreact.